src/neural/blas/transforms.cc

```diff
@@ -258,22 +256,22 @@ void Transforms::WinogradConvolve3(const int outputs,
   WinogradTransformOut(M, output, outputs);
 }
 
-template <unsigned int filter_size>
```

Comment:
Why would the filter size ever change from 1?

Reply:
This code is very much dead. It has not been modified once since Gary first made the port from Leela Go. I'm not even sure it's changed in Leela Zero's history, ever.